Doc/tools/templates/download.html

```diff
@@ -1,10 +1,6 @@
 {% extends "layout.html" %}
 {% set title = 'Download' %}
-{% if daily is defined %}
-  {% set dlbase = pathto('archives', 1) %}
-{% else %}
-  {% set dlbase = 'https://docs.python.org/ftp/python/doc/' + release %}
-{% endif %}
+{% set dlbase = pathto('archives', 1) %}
 
```

Comment:
This change should not be made. The non-daily path is used for the archived documention set for each release (cataloged at https://www.python.org/doc/versions/). It is a minor wart that the generated links do not exist prior to the first alpha release in a feature release; the 404's will go away after 3.12.0a1 and the on-line versions of the doc are correct.

Reply:
Thank you. Reverted and added a comment to prevent future removal attempts.